components/Layout/HomeHeader/HomeHeaderItems.vue

```diff
@@ -10,6 +10,22 @@
     >
       <swiper-slide v-for="item in items" :key="item.Id">
         <div class="slide-backdrop">
+          <div
+            v-if="
+              !item.BackdropImageTags ||
+              (item.ImageTags && !item.ImageTags.Backdrop)
+            "
+            class="default-icon"
+          >
+            <v-icon
+              :size="$vuetify.breakpoint.mdAndUp ? 256 : 128"
+              class="text--disabled"
+              color="white"
+              dark
+            >
+              {{ getItemIcon(item) }}
+            </v-icon>
+          </div>
           <blurhash-image
             :key="`${item.Id}-image`"
```

Comment:
v-else?

Reply:
It's waiting on the refactoring of blurhash-image and such by ferferga, since not all the conditions are covered by the v-if above.

Reply:
If you checked against my branch forget this but, if not, I don't know if that would work, as the blurhash image component checks for both canvas and image. If canvas is not available, it won't create the component, but it will create the image.

We can merge both and check all the cases and do another PR though